src/client/nav/nav.js

```diff
@@ -415,7 +415,8 @@ spf.nav.navigate_ = function(url, options, info) {
       spf.nav.navigateAddHistory_(url, info.referer, handleError);
     }
     // Then attempt to scroll.
-    spf.nav.navigateScroll_(url, info.position);
+    spf.nav.navigateScroll_(url, info.position, true);
+    info.scrolled = true;
```

Comment:
Do you think it makes sense to pass info into `navigateScroll_` and then have that set `info.scrolled = true`?